File: Assets/PlayerShip.cs
using UnityEngine;
using System.Collections;

public class PlayerShip : MonoBehaviour {
	bool moving = false;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
		if (moving) {
			Vector3 moveDir = new Vector3(0, 1, 0);
			float speed = 5;
	    	transform.Translate(speed * moveDir * Time.deltaTime);
		}
	}
	
	[RPC]
	public void ToggleMoving() {
		Debug.Log ("toggling");
		moving = !moving;
	}
}

File: Assets/PlayerShip.js
#pragma strict

var moving = false;
var helmOccupied = false;

	// Use this for initialization
function Start () {

}

// Update is called once per frame
function Update () {
	if (moving) {
		var moveDir = Vector3(0, 0, 1);
		var speed = 5;
    	transform.Translate(speed * moveDir * Time.deltaTime);
	}
	
	// proof of concept for sending ship data to clients
	if (Network.peerType == NetworkPeerType.Server) {
		var go:GameObject = GameObject.Find("CommandRelay"); 
		var bm:CommandRelay = go.GetComponent(CommandRelay);
		go.networkView.RPC("fromShipToRelay", RPCMode.AllBuffered);
	}
}

@RPC
function ToggleMoving() {
	moving = !moving;
}

@RPC
function Fire() {
	Debug.Log("Firing");
}